packages/@aws-cdk/aws-rds/test/test.cluster.ts

```diff
@@ -1125,7 +1259,7 @@ export = {
     // WHEN
     new DatabaseCluster(stack, 'Database', {
       engine: DatabaseClusterEngine.auroraPostgres({
-        version: AuroraPostgresEngineVersion.VER_11_4,
+        version: AuroraPostgresEngineVersion.VER_11_6,
```

Comment:
Why this change here...?

**EDIT**: ah, is it because `11.4` in fact doesn't support one of the features used in the test...? 🙂

Reply:
yep!